mathesar/models.py

```diff
@@ -127,6 +133,19 @@ class Table(DatabaseObject):
                                related_name='tables')
     import_verified = models.BooleanField(blank=True, null=True)
 
+    def validate_unique(self, exclude=None):
```

Comment:
I would call this `_validate_unique` to indicate that it's an internal method.

Reply:
[`validate_unique`](https://docs.djangoproject.com/en/3.2/ref/models/instances/#django.db.models.Model.validate_unique) is actually a defined django method. I think it is most often used in the context of a form, where you call `full_clean`, but still seems somewhat idiomatic here. Happy to change if you prefer the internal method.

Reply:
Oh, it is? Let's use it then.